runner/src/integrationTest/kotlin/com/cosmotech/runner/service/RunnerServiceIntegrationTest.kt

```diff
@@ -199,20 +201,21 @@ class RunnerServiceIntegrationTest : CsmRedisTestBase() {
 
   @Test
   fun `test CRUD operations on Runner as Platform Admin`() {
-    every { getCurrentAuthenticatedRoles(any()) } returns listOf("Platform.Admin")
+    every { getCurrentAccountIdentifier(any()) } returns "random_user_with_patform_admin_role"
+    every { getCurrentAuthenticatedRoles(any()) } returns listOf(ROLE_PLATFORM_ADMIN)
 
     logger.info("should create a second Runner")
-    val runner2 =
+    val runner3 =
```

Comment:
There are at least 2 other places in this test that should be updated form 2 to 3 (the log above `a second Runner` and the variable below `runnerSaved2`.
But the best thing to do is probably to update the test to be less tied to what is done exactly in the other tests:
* list and count the current number of runner at the start of the test
* do the test and assert there is `+ 1` runner after the create and back to the original count after the delete
* genericize logs (e.g. "create another Runner",...) and names ("AnotherRunner" or whatever)

Reply:
it's log info mistake, data is wipe every test. I'll change log to match what's actually done.